torch/_torch_docs.py

```diff
@@ -6994,8 +6994,8 @@ def merge_dicts(*dicts):
 
 Arguments:
     condition (BoolTensor): When True (nonzero), yield x, otherwise yield y
-    x (Tensor): values selected at indices where :attr:`condition` is ``True``
-    y (Tensor): values selected at indices where :attr:`condition` is ``False``
+    x (Tensor or Scalar): values selected at indices where :attr:`condition` is ``True``
```

Comment:
I'm not sure if this is covered in docs elsewhere or what's the standard way of describing scalars

If the user sees is for the first time, it may be not clear if only PyTorch scalars meant, or if Python number scalars (and again current type promotion story should be somehow mentioned?) are okay as well? At least examples of what currently works and what does not, would be useful

Reply:
Makes sense. Would be totally worth adding an example. Thanks!